### PR TITLE
[plugins/whois] Add support for multi-domain check

### PR DESCRIPTION
**Pull request by ruliane**
This PR adds support for checking several domains in one plugin, through the DOMAIN environment variable. Plugin is renamed from whois_ to whois, to reflect this change.
It also changes the shellcheck when importing plugin.sh (formerly SC1090, now SC1091).

### CODE REVIEW

File: plugins/network/whois [hunk 1]
```diff
@@ -6,35 +6,37 @@
 
 =head1 NAME
 
-whois_ - Plugin to monitor expiry dates of domain in the WHOIS database
+whois - Plugin to monitor expiry dates of domain in the WHOIS database
```
Comment (sumpfralle) [inline plugins/network/whois:9]: Maybe *of domains* (instead of *of domain*)?

File: plugins/network/whois [hunk 2]
```diff
@@ -78,43 +80,48 @@ EOF
...
 	local FIELDNAME
 	local CACHEFILE
 
-	FIELDNAME="$(clean_fieldname "${NAME}")"
+	for NAME in $DOMAINS
+	do
+		FIELDNAME="$(clean_fieldname "${NAME}")"
 
-	CACHEFILE="${MUNIN_PLUGSTATE}/$(basename "${0}").${FIELDNAME}.cache"
+		CACHEFILE="${MUNIN_PLUGSTATE}/$(basename "${0}").${FIELDNAME}.cache"
```
Comment (sumpfralle) [inline plugins/network/whois:108]: I did not look at the new file, but the github visualization indicates, that you are mixing tabs and spaces here (and in some more lines below)?

**Review by sumpfralle — COMMENTED** [review]
That looks like a good improvement of the plugin!

Please take a look at my minor comments below. Thanks!

File: plugins/network/whois (hunk 1 above)
Comment (ruliane) [reply plugins/network/whois]: Right, now fixed.

File: plugins/network/whois (hunk 2 above)
Comment (ruliane) [reply plugins/network/whois:108]: Nope, only tabs are used. I included existing code in a `for` loop and GitHub highlights tabs added for indentation.

**Comment by sumpfralle** [comment]
Thank you!

**Pull request merged**